src/test/java/bio/terra/service/journal/JournalServiceTest.java

```diff
@@ -1,283 +1,193 @@
 package bio.terra.service.journal;
 
-import static bio.terra.service.journal.JournalService.getCallerFrame;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.empty;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.equalToIgnoringCase;
-import static org.hamcrest.Matchers.hasSize;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
-import bio.terra.common.EmbeddedDatabaseTest;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.fixtures.AuthenticationFixtures;
 import bio.terra.common.iam.AuthenticatedUserRequest;
-import bio.terra.model.JournalEntryModel;
 import bio.terra.service.auth.iam.IamResourceType;
-import bio.terra.service.job.JobMapKeys;
-import java.util.LinkedHashMap;
-import java.util.List;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.UUID;
-import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.dao.DataIntegrityViolationException;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringRunner;
 
-@RunWith(SpringRunner.class)
-@AutoConfigureMockMvc
-@SpringBootTest
 @ActiveProfiles({"google", "unittest"})
+@ExtendWith(MockitoExtension.class)
 @Tag("bio.terra.common.category.Unit")
-@EmbeddedDatabaseTest
 public class JournalServiceTest {
```

Comment:
Previously, this test would be more accurately categorized as a connected test: it called an autowired `JournalService` with all of its real dependencies, including `JournalDao`.  Much of what was being tested was `JournalDao` code.

So I broke out most of the existing testing into a new `JournalDaoTest` with an embedded database, and made this test class a unit test with mocking that tests `JournalService`.

Reply:
I ended up reverting refactors of embedded database tests to JUnit 5.  From the look of things, they were running 4x slower than their original counterparts. Possible explanations:

- JUnit 5 versions are legitimately running 4x slower than JUnit 4 versions (possibly a change in configuration or annotation is required to work with Zonky in JUnit 5 tests)
- JUnit 4 versions are not reporting runtime in the way I'd expect in Gradle results / scans (is initialization excluded?)
- JUnit 4 versions are not behaving as we'd expect with respect to embedded databases (possibly not performing the initialization we expect)

I don't want to risk driving up unit test runtime while we don't understand how to configure JUnit 5 embedded database tests in a performant, reliable way.